Example/Example/Purchase/CartViewController.swift

```diff
@@ -93,24 +114,62 @@ final class CartViewController: UIViewController, UITableViewDataSource {
       tableView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
       tableViewBottomAnchor,
     ])
+
+    let editCheckoutButton = UIBarButtonItem(
+      title: "Edit",
+      style: .plain,
+      target: self,
+      action: #selector(presentPickerController)
+    )
+    navigationItem.setRightBarButton(editCheckoutButton, animated: false)
   }
 
   override func viewDidLoad() {
     super.viewDidLoad()
-
     eventHandler(.cartDidLoad(self.cashButton))
+
+    updateViewState()
+  }
+
+  // MARK: - Private
+
+  func updateViewState() {
+    switch checkoutOption {
+    case .v1Checkout:
+      checkoutTypeTextField.text = "Checkout V1"
+      cashButton.isHidden = true
+    case .v2Checkout:
+      checkoutTypeTextField.text = "Checkout V2"
+      cashButton.isHidden = false
+    case .v3Checkout:
+      checkoutTypeTextField.text = "Checkout V3"
+      cashButton.isHidden = false
+    }
   }
 
   // MARK: Actions
 
   @objc private func didTapPay() {
-    eventHandler(.didTapPay)
+    switch checkoutOption {
+    case .v1Checkout:
+      print("TODO - FIXME")
```

Comment:
@ScottAntonacAP how can I present V1 checkout?

Reply:
Currently on `main` branch, the express checkout switch in the example app is a actually a V1 / V2 switch. What it should be doing is setting the checkout mode.

That switch (just the express switch) could potentially be removed for now and the functionality moved to the new buttons created in this PR.
 
It _currently_ works as follows:

1. Executing `didTapPay` in `PurchaseFlowController` calls `logicController.payWithAfterpay()`
2. `payWithAfterpay()` checks if express is enabled. if not, run the `showAfterpayCheckoutV1` command with the retrieved checkout url
3. The command from step 2 runs `Afterpay.presentCheckoutModally()` which is the SDK method

In the future functionality could be added to V2 to use Express OR Standard checkout.